crates/fuzzing/Cargo.toml

```diff
@@ -43,4 +47,12 @@ rand = { version = "0.8.0", features = ["small_rng"] }
 wasm-spec-interpreter = { path = "./wasm-spec-interpreter", optional = true, features = ['build-libinterpret'] }
 
 [features]
+default = ['fuzz-spec-interpreter']
```

Comment:
Is this necessary for this PR? (I think this might affect `cargo fuzz build --no-default-features` as still trying to build the spec interpreter)

Reply:
Not sure where that came from, honestly.  I should be able to remove it.